canonical-web-and-design/maas-ui
# Add SSH keys

- Add a form to add SSH keys. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1504. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1503. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1505.

QA:
- Navigate to the SSH key list in your profile.
- Click "Import SSH key".
- Try importing a GitHub and Launchpad key and paste a local key.
- Each key should be created and appear in the list.
